### PR TITLE
[FIX] Refactor playlist event : Fix Track count

### PR DESCRIPTION
**Pull request by flyingrub**
…ed from a playlist

### CODE REVIEW

**Comment by flyingrub** [comment]
@aenario this PR will probably conflict with another PR that will allow to sort by title / album. It would be great if you could review this one in priority. :)

File: app/models/playlist.js [hunk 1]
```diff
@@ -57,20 +59,61 @@ const Playlist = Backbone.Model.extend({
         }
     },
 
+    bindListeners() {
+        this.on('change:hidden', this.removeTrack, this);
+
+        // AllTracks doesn't need all other listener
+        if (this.get('type') == 'all') return;
```
Comment (nono) [inline app/models/playlist.js:66]: Your linter don't ask for `===`?

File: app/models/playlist.js [hunk 2]
```diff
@@ -57,20 +59,61 @@ const Playlist = Backbone.Model.extend({
...
+        if (this.get('type') == 'upNext') {
+            // Add the current Track to upNext
+            this.listenTo(application.appState,
+                'change:currentTrack',
+                function(appState, currentTrack) {
+                    this.addTrack(currentTrack);
+                }
+            );
+            this.listenTo(
+                application.channel,{
```
Comment (nono) [inline app/models/playlist.js:77]: A space between `,` and `{` would be nice

**Comment by nono** [comment]
Looks good to me!

**Pull request merged**